react-native/react/update/index.render.desktop.js

```diff
@@ -0,0 +1,172 @@
+import React, {Component} from '../base-react'
+import {Checkbox, FlatButton} from 'material-ui'
+import Snackbar from 'material-ui/lib/snackbar'
+import commonStyles, {colors} from '../styles/common'
+import Header from '../common-adapters/header'
+import {clipboard} from 'electron'
+import resolveAssets from '../../../desktop/resolve-assets'
+import marked from 'marked'
+import {resizeWindow} from '../native/remote-component-helper'
+
+export default class UpdateRender extends Component {
+  constructor (props) {
+    super(props)
+
+    this.state = {
+    }
+  }
+
+  componentDidMount () {
+    resizeWindow()
+  }
+
+  componentDidUpdate () {
+    resizeWindow()
+  }
+
+  onCopy () {
+    clipboard.writeText(this.props.updateCommand)
+    this.refs.clipboardSnackbar.show()
+  }
+
+  render () {
+    const realCSS = `
+      .clipboard { color: #A29E9E; }
```

Comment:
here we're injecting real css as we need :hover we could discuss using radium or some other solution